lib/src/characters_impl.dart

```diff
@@ -116,13 +116,14 @@ class StringCharacters extends Iterable<String> implements Characters {
   }
 
   @override
-  bool contains(Object? other) {
-    if (other is String) {
-      if (other.isEmpty) return false;
-      var next = Breaks(other, 0, other.length, stateSoTNoBreak).nextBreak();
-      if (next != other.length) return false;
+  bool contains(Object? element) {
+    if (element is String) {
+      if (element.isEmpty) return false;
+      var next =
+          Breaks(element, 0, element.length, stateSoTNoBreak).nextBreak();
+      if (next != element.length) return false;
       // [other] is single grapheme cluster.
```

Comment:
Comment's "other" refers to "element" too, and should be renamed as well

Reply:
Done